README.md

```diff
@@ -1,35 +1,108 @@
-# Azure Functions PowerShell New Programming Model (Extension Bundles)
+# Azure Functions PowerShell Insider Preview of the v2 Programming Model
 
-This repository contains skeleton code for Azure PowerShell Functions in the V2 programming model.
+This repository contains skeleton code for Azure Functions using the V2 programming model for PowerShell.
 
 ---
+# Getting started
 
-## Installation Instructions
+## Prerequisites
+
+* An Azure account with an active subscription.[ Create an account for free.](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio)
+* Private build of Azure Functions Core Tools (Instructions below)
+* [PowerShell 7.2](https://learn.microsoft.com/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.2#msi)
+* [.Net Core 6 version SDK 6.0.408 or higher](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) 
+* [.Net Core 8 version SDK 8.0.100-preview.3 or SDK 8.0.100-preview.4](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) **Please note that though this version is currently in preview, it is required for using the v2 PowerShell programming model.**
```

Comment:
Let's explicitly state (in bold) that preview.5 has a known compatibility issue, so they shouldn't even try to use it.